test-app/app_nrf52.c

```diff
@@ -43,6 +43,10 @@
 #define UART0_TXD_MAXCOUNT *((volatile uint32_t *)(UART0_BASE + 0x548))
 #define UART0_BAUDRATE     *((volatile uint32_t *)(UART0_BASE + 0x524))
 
+
+__attribute__((section(".rodata")))
```

Comment:
Wrap with `__GNUC__`?

Reply:
actually this is not needed (default section for const variables is .rodata). For some reason, the app includes this only when DEBUG=1